www/install.rst

```diff
@@ -6,13 +6,12 @@ These are instructions for installing :ref:`the full SciPy stack
 <stackspec>`.  For installing individual packages, such as NumPy and
 SciPy, see :ref:`individual-packages` below.
 
-
 Scientific Python distributions
 -------------------------------
 
-For most users, especially on Windows and Mac, the easiest way to install the
-packages of the SciPy stack is to download one of these Python distributions,
-which includes all the key packages:
+For most users, especially on Windows, the easiest way to install the packages
```

Comment:
I agree that the situation on OS X is hugely better than on Windows. I just want to ensure that it's very clear to every new user that comes in, he/she should use a distribution. Actually, I should correct myself: Anaconda or Miniconda. It is the best choice, for many reasons. The text ``For most users, especially on Windows ...`` doesn't really do that well enough.

There's an issue with us recommending one over the others, but on the other hand we do new users a big disservice if we don't point them to the best option. 

With ``conda`` and ``conda-forge`` we now have things that are fully open source and community-oriented. Oriented on *our* community, unlike ``pip`` and ``PyPI``. I see the differences in usability and capabilities only getting bigger, looking at how fast ``conda`` and ``conda-forge`` move and how disfunctional decision making for ``pip`` is.

> We tested the hell out of the numpy and scipy wheels 

I know, and your efforts are much appreciated! The situation has improved massively. 

> do not directly support the Anaconda / Canopy etc packages

Actually, we do for conda-forge. I'm one of the maintainers for ``numpy`` and ``scipy`` now, and many other contributors and core devs are involved for those and other packages.